Documentation/BuildInstructionsLadybird.md

```diff
@@ -94,13 +94,36 @@ sudo dnf install autoconf-archive automake ccache cmake curl git libdrm-devel li
 ### openSUSE:
 
 ```
-sudo zypper install autoconf-archive automake ccache cmake curl gcc14 gcc14-c++ git liberation-fonts libglvnd-devel nasm ninja qt6-base-devel qt6-multimedia-devel qt6-tools-devel qt6-wayland-devel tar unzip zip
+sudo zypper install autoconf-archive automake ccache cmake curl gcc14 gcc14-c++ git liberation-fonts libglvnd-devel libtool nasm ninja qt6-base-devel qt6-multimedia-devel qt6-tools-devel qt6-wayland-devel tar unzip zip
 ```
 
-It is currently recommended to install the `libpulse-devel` package to avoid runtime dynamic linking issues
+If during the build process one or more of the base repository packages are flagged as not up to the required version, for example, on Leap 15.6 the autoconf package might be at 2.69, but an error message indicates that to build gperf requires 2.70. To get the needed version installed may require adding some additional repository such as the experimental `devel:tools:building`.
+
+As an example for Leap 15.6 (other releases would need to change the specific command used, do some searches online to get the most up to date information)
```

Comment:
The wording here is rather grammatically incorrect and confusing to read. How about:

> If one or more of the base repository packages are flagged as having an out-of-date version during the build process, you may need add the `devel:tools:building` repository. For example, on Leap 15.6, the `autoconf` package might be version 2.69, whereas the `gperf` package requires 2.70 to build.
>
> For Leap 15.6, run the following command to add this repo (you will need to adjust the URL for different Leap versions):
> ```
> sudo zypper addrepo https://download.opensuse.org/repositories/devel:tools:building/15.6/devel:tools:building.repo
> sudo zypper refresh
> ```

Reply:
@trflynn89 I used almost everything word for word.  I made some minor changes to the last sentence. Thank you!